### ISSUE TITLE
Show remaining time until site adding completes (as a progress bar)

### ISSUE THREAD

**Issue by narc0tiq** [narc0tiq/YARM#5]
When adding a site, there is a ten second grace period during which you can add more entities of the same ore type (e.g., for merging two shards of the same kind of ore, or for oil sites). We should show visual feedback while this timer counts down, to allow the player to gauge how quickly they need to move.

**Comment by jocker-il** [comment]
A very good idea ! I second that

**Comment by georgehank** [comment]
Yes, this.  Since there's no (is there?) way to shorten that interval... for adding sort-of close by fields that you do want as separate fields anyway.

**Comment by narc0tiq** [comment]
With the UI refactor coming in 1.0, there's probably a chance to add this. It's not that hard, just check if `player_data.current_site` and we can further check for a `progress { total = 120, started_at = --[[tick number]] 12345 }` table, and that should be enough to make a time-progress bar.
